partner_noncommercial/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 Therp BV (http://therp.nl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Partner Non Commercial',
+    'version': '10.0.1.0.0',
+    "author": "Therp BV,Camptocamp,Odoo Community Association (OCA)",
```

Comment:
Add Website as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)